sql/sys_vars.cc

```diff
@@ -1570,6 +1570,15 @@ Sys_log_bin_compress_min_len(
   GLOBAL_VAR(opt_bin_log_compress_min_len),
   CMD_LINE(OPT_ARG), VALID_RANGE(10, 1024), DEFAULT(256), BLOCK_SIZE(1));
 
+static Sys_var_enum Sys_binlog_error_action(
+       "binlog_error_action",
+       "When statements cannot be written to the binary log due to a fatal "
+       "error, the server can either ignore the error and let the primary "
+       "continue, or abort",
+       GLOBAL_VAR(binlog_error_action), CMD_LINE(REQUIRED_ARG),
+       (const char *[]){"IGNORE_ERROR", "ABORT_SERVER", NullS},
```

Comment:
I felt that it’s excessive to exile the enum list at a different place but still only use it here.

Reply:
Darn. Compount literals is a GCC extension for C++, even though it’s [standardized in C99](https://en.cppreference.com/w/c/language/compound_literal), [our regular-C version](https://github.com/MariaDB/server/blob/main/CODING_STANDARDS.md#language-standards).